Index: obit/python/UV.inc
===================================================================
--- obit.orig/python/UV.inc
+++ obit/python/UV.inc
@@ -36,6 +36,7 @@
 #include "ObitUVUtil.h"
 #include "ObitUVEdit.h"
 #include "ObitUVWeight.h"
+#include "ObitTableCLUtil.h"
 #include "ObitTableSNUtil.h"
 #include "ObitTableNXUtil.h"
 #include "ObitTableFG.h"
Index: obit/python/Obit_wrap.c
===================================================================
--- obit.orig/python/Obit_wrap.c
+++ obit/python/Obit_wrap.c
@@ -17144,6 +17144,7 @@ extern long UVImagerIsA (ObitUVImager* i
 #include "ObitUVUtil.h"
 #include "ObitUVEdit.h"
 #include "ObitUVWeight.h"
+#include "ObitTableCLUtil.h"
 #include "ObitTableSNUtil.h"
 #include "ObitTableNXUtil.h"
 #include "ObitTableFG.h"
